insights/specs/sos_archive.py

```diff
@@ -63,6 +63,7 @@ class SosSpecs(Specs):
     ip_s_link = first_of([simple_file("sos_commands/networking/ip_-s_-d_link"), simple_file("sos_commands/networking/ip_-s_link"), simple_file("sos_commands/networking/ip_link")])
     iptables = first_file(["/etc/sysconfig/iptables", "/etc/sysconfig/iptables.save"])
     journal_since_boot = first_of([simple_file("sos_commands/logs/journalctl_--no-pager_--boot"), simple_file("sos_commands/logs/journalctl_--no-pager_--catalog_--boot")])
+    libvirt_qemu_log = glob_file(r"/var/log/libvirt/qemu/*.log")
```

Comment:
For the `*_file` specs, it's not necessary to add them into the `sos_archive.py` or `insights_archive.py`, if the file_path is the same in different archives, the one in `default.py` works for all of them.

Reply:
Got it.